src/_common/lightbox/lightbox.styl

```diff
@@ -31,48 +30,69 @@ $-controls-padding = 10px
 		background-color: rgba(0, 0, 0, 0.85)
 
 .-controls
-	flex: 0 0 auto
-	height: $-controls-height
-	line-height: $-controls-height
-	text-align: center
+	z-index: 1
 
-	@media only screen and (orientation: landscape)
-		position: absolute
-		right: $-controls-padding
-		bottom: 0
-		z-index: 10
+	&-general
+		flex: 0 0 auto
+		height: $-controls-height
+		line-height: $-controls-height
+		text-align: center
+		transition: right 300ms ease
 
-	@media $media-sm-up
-		text-align: right
-		position: static
-		padding-right: $-controls-padding
+		@media only screen and (orientation: landscape)
+			position: absolute
+			right: $-controls-padding
+			bottom: 0
+			z-index: 10
+
+			&.-slider-showing
+				right: $-controls-padding + 30px
+
+		@media $media-sm-up
+			text-align: right
+			position: static
+			padding-right: $-controls-padding
+
+	&-slider
+		transition: opacity 500ms
+		transition-timing-function: ease
 
-.-next, .-prev
-	rounded-corners-lg()
-	display: none
+		&.-hide
+			opacity: 0
+			transition-timing-function: $strong-ease-out
+
+.-next
+.-prev
 	position: absolute
-	top: 50%
-	width: $-button-size
-	height: $-button-size
+	top: 0
+	width: $-button-size-xs
+	height: 100%
 	line-height: $-button-size
-	margin-top: -(@height / 2)
 	z-index: 1
 	background-color: rgba($black, 0.5)
-	text-align: center
+	display: flex
+	justify-content: center
+	align-items: center
 
 	@media $media-sm-up
-		display: block
+		rounded-corners-lg()
```

Comment:
I think we should instead float the next/prev controls to the bottom on mobile so that it doesn't take up space from left/right. Maybe we should just disable the lightbox dragging on mobile if it's not really working well?

Reply:
I moved the controls to the bottom for phones, but didn't change the dragging functionality.